utility/src/main/java/org/oppia/android/util/logging/ConsoleLogger.kt

```diff
@@ -124,10 +128,23 @@ class ConsoleLogger @Inject constructor(
   }
 
   /**
-   * Writes the specified text line to file in a background thread to ensure that saving messages don't block the main
-   * thread. A blocking dispatcher is used to ensure messages are written in order.
+   * Writes the specified text line to file in a background thread to ensure that saving messages
+   * doesn't block the main thread. A blocking dispatcher is used to ensure messages are written
+   * in order.
    */
   private fun logToFileInBackground(text: String) {
-    blockingScope.launch { logDirectory.printWriter().use { out -> out.println(text) } }
+    blockingScope.launch {
+      if (printWriter == null) {
+        printWriter = PrintWriter(FileWriter(logDirectory, true)) // Open in append mode.
+      }
+      printWriter?.println(text)
+      printWriter?.flush()
+    }
+  }
+
+  /** Close the log file when logging is finished. */
+  private fun closeLogFile() {
+    printWriter?.close()
+    printWriter = null
```

Comment:
Follow up -- seems this was only used in tests before but it is nolonger needed. Please confirm and remove.

Reply:
consoleLogFile removed.